SDKGenerator.njsproj

```diff
@@ -101,6 +101,11 @@
     <OutputPath>bin\</OutputPath>
     <ScriptArguments>-destpath ..\sdks\XPlatCppSdk -apiSpecGitUrl -buildIdentifier XPlatCppSdk_manual</ScriptArguments>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'XPlatCSdk' ">
+    <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\XPlatCSdk\</OutputPath>
+    <ScriptArguments>-destpath ..\..\src\playfabcorec\playfabcorecsdk -apiSpecGitUrl -buildIdentifier playfabcorecsdk_manual</ScriptArguments>
```

Comment:
>..\..\src\playfabcorec\playfabcorecsdk [](http://example.com/codeflow?start=31&length=38)

This seems like a strange thing to hardcode into the build config, but I see that its been done that way for the other configurations as well.  Is there a way to override this at build time?

I'd prefer the directory structure to be something like this:

src\PlayFabCore\
  PlayFabCoreCSdk
  PlayFabCoreCSdk_Template

I think its a bit more readable and matches what we've got for the github repo names

Reply:
Yeah the path is kinda gross. That's why I was wondering earlier if we'd already had some system to hook down into here given it's a submodule. I'm not sure what you're thinking of with "override this at build time". If we can instead write some kind of build script at the template level that procs SDKGenerator and passes in all the needed paths, that would be nice. It would probably completely remove the need for any change here.

I've got no problem with changing those directory names.

Reply:
To be clear, I intended to explore that build script option a bit before I realized the hardcoded path is apparently the standard design here.

Reply:
There are some build scripts in /SDKBuildScripts/

Maybe would be a good idea to add one for Core C SDK there (in addition to adding this VS build config probably)

Reply:
Changed folder names. Build script is being modified at the template level as discussed in teams.